tests/parsers/test.locale-messagesjson.js

```diff
@@ -120,11 +118,27 @@ describe('LocaleMessagesJSONParser', () => {
   }
 }`, addonsLinter.collector);
     localeMessagesJSONParser.parse();
-    expect(localeMessagesJSONParser.isValid).toEqual(false);
+    expect(localeMessagesJSONParser.isValid).toEqual(true);
     const { errors } = addonsLinter.collector;
-    expect(errors.length).toEqual(1);
-    expect(errors[0].code).toEqual(messages.INVALID_PLACEHOLDER_NAME.code);
-    expect(errors[0].message).toEqual(messages.INVALID_PLACEHOLDER_NAME.message);
+    expect(errors.length).toEqual(0);
+  });
+
+  it('should support dot in placeholder name', () => {
```

Comment:
The dot is not a valid character in the placeholder names and it will be ignored, and so this additional test should just be removed.